recipes/yajl/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir, rm, rename
```

Comment:
There are no patches

```suggestion
from conan.tools.files import get, copy, rmdir, rm, rename
```